_posts/2020-08-05-security-of-nano.md

```diff
@@ -0,0 +1,81 @@
+
+============================
+## The Nano Protocol
+The blocks are validated and voted on in a distributed consensus model across the world using network nodes. If the nodes agree a block is valid, it's stored in the Ledger which is synchronized between the nodes.
+
+The security of your Nano is bound to the security of the network. Without a functioning network, the Nano can't be transferred and the value of the network (the value of Nano) would eventually drop to zero. That topic is too broad for this article and we will simply assume the network is functioning. There is plenty of material available as it's been discussed for years but easiest would be to refer to the successful [official security audit that was made](https://medium.com/nanocurrency/nano-protocol-security-audit-summary-and-full-report-48760be8ab3d) in 2018. The network has additonally been heavily improved since then.
+
+============================
+## The Storage of Keys
+There are many ways to store keys/seeds and this is where Wallets come in. A wallet can be digital, analogue, online, offline, encrypted or open. How YOU decide to store the seed is often bound to the level of convenience you are after. That is further described in [this article](https://medium.com/nano-education/how-to-secure-your-cryptocurrency-nano-7a83b194e474). We will be focusing on storing the keys in Nault, a hardware wallet or offline using the [remote-signing procedure](#transact-using-a-remote-signing-procedure).
+
+### Keys in Nault
+Nault is a fully client-side wallet meaning it will never send the keys to a remote server. They are stored only in memory or in the browser cache depending on how to set it up. The default usage in Nault is to encrypt the seed using a password of your choice. The strength of the password will decide the level of security and it will be stored as long as the browser cache lives, which can be forever if you want. Nault can also be set to remove all wallet data on browser exit. Then you will enter your seed or key the next time you need access. Using that method probably means you would store the key on a paper and scan it using the QR reader. Having the key in a password manager is not recommended. Nault can be used Online or Offline which is described in the [the usage of keys](#the-usage-of-keys) section. You can for example store the keys in Nault but on a machine that is always offline.
```

Comment:
Stored in the browsers local storage.

Why is a password manager not recommended?

Reply:
Will change both. What I meant was a password manager does not add any security since Nault also encrypt using a password. Better to store key offline if anything.